setup.cfg

```diff
@@ -119,6 +119,7 @@ filterwarnings =
     ignore:unclosed <socket:ResourceWarning
     ignore:unclosed <ssl.SSLSocket:ResourceWarning
     ignore:numpy.ufunc size changed:RuntimeWarning
+    ignore:numpy.ndarray size changed:RuntimeWarning
```

Comment:
I suppose it appears when several binary wheels compiled against different NumPy versions interact? The oldest reference I can find is https://github.com/numpy/numpy/issues/563, but there's not that much information available.

Reply:
In this case, it may be definitions about `ndarray` inside cython. I do think that, like for the `dtype`, the extension of the struct that holds the definition of an array is written such that one can always use the old version.

Reply:
Could the changes on `PyArrayObject` affect us in some way ?
https://mail.python.org/pipermail/numpy-discussion/2020-November/081213.html

Reply:
Will ignoring this have any side effects in the future? When can we remove ignoring this warning?

Reply:
I think our code suffers only because we use cython, so at some level it is not our problem... And the thread does suggest that it it is fine to have code that relies on the old `struct`. So, for now I think we can simply do this...

As for when we can remove the warning: presumably whenever we have a minimum version of cython compatible with this change...

Reply:
Ok, fine by me then :+1: